### PR TITLE
StyleIndex

### PR DESCRIPTION
**Pull request by SelcoD**

### CODE REVIEW

File: components/EditTripButton.js [hunk 1]
```diff
@@ -1,3 +1,30 @@
+import styled from "styled-components";
+
+const EditButton = styled.button`
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  padding: 50px; /* Adjust the padding to increase the button size */
```
Comment (NilsOt1) [inline components/EditTripButton.js]: you could delete the comment

File: components/TripList.js [hunk 2]
```diff
@@ -24,19 +31,67 @@ export default function TripList({ campingTrips, onDeleteTrip }) {
...
+  background-color: #e66465;
+  color: black;
+  border: none;
+  padding: 10px;
+  border-radius: 4px;
+  font-size: 16px;
+  cursor: pointer;
+  // display: flex;
+  // align-items: center;
+  // justify-content: center;
```
Comment (NilsOt1) [inline components/TripList.js]: You could delete the comments here aswell

**Review by NilsOt1 — APPROVED** [review]
Just two small issues. Good job! 🚀

**Comment by SelcoD** [comment]
I forgot to create a new branch, so thats my "feature/StyleForm" for now.

**Review by MonaHer — COMMENTED** [review]
LGTM :) Good job, Herr Drapalla 🚀✨

**Pull request merged**